docs/how_to/keys/grok.md

```diff
@@ -0,0 +1,106 @@
+
+<p align="center">
+    <img src="../assets/grok_00_main_page.png" alt="Grok AI" width="500"/>
+</p>
+
+!!! info
+
+    If you already have an account, go ahead and skip ahead to [Step 2](#2-set-up-billing)
+
+### 1. Create a Grok account
+
+1. Click on the console login option, or navigate to [https://accounts.x.ai/sign-up](https://accounts.x.ai/sign-up)
+
+    <p align="center">
+    <img src="../assets/grok_01_console_login.png" alt="Console Login" width="500"/>
+    </p>
+
+1. Complete the signup process, it is extremely straightforward.
+
+    <p align="center">
+    <img src="../assets/grok_02_signup.png" alt="Signup" width="500"/>
+    </p>
+
+### 2. Set Up Billing
+
+!!! warning "Billing Required"
+
+    Before using Grok models with Griptape Nodes, be aware that Grok.ai _requires_ billing information to be set up. Without this step, the models won't be available for use in your Nodes.
+
+    If you attempt to run Nodes using Grok without completing this billing setup, your workflows will fail as the service will reject the account credentials.
+
+<p align="center">
+    <img src="../assets/grok_03_billing.png" alt="Billing" width="600"/>
+</p>
+
+### 3. Generate an API Key
+
+1. Navigate to the API keys section; there are multiple links to take you there.
+
+    <p align="center">
+    <img src="../assets/grok_04_key_links.png" alt="API Key links" width="600"/>
+    </p>
+
+1. Click on "Create API Key"
+
+    <p align="center">
+    <img src="../assets/grok_05_create_api_key_button.png" alt="Create API Key Button" width="500"/>
+    </p>
+
+1. Name your API key (suggested name: GriptapeNodes!)
+
+    <p align="center">
+    <img src="../assets/grok_06_create_api_key_page.png" alt="Create API Key Page" width="500"/>
+    </p>
+
+1. Click **save**
+
+1. Copy and securely store your API key
+
+    !!! danger "Security Notice"
+
+        It is recommended to save this token in a password locker or secure notes app, so you can find it, but also keep it secure.
+
+        Your access token is a personal identifier for requests made to OpenAI services. Never share it with anyone, and take precautions to avoid displaying it in screenshots, videos, or during screen-sharing sessions.
+
+        Treat it like you would a credit card number.
+
+## Add Your API Key to Griptape Nodes Settings
+
+!!! info "Overview"
+
+    Now that you've set up your Grok account, you need to configure Griptape Nodes to use your API key. This process is straightforward.
+
+### 1. Open the Griptape Nodes Settings Menu
+
+1. Launch Griptape Nodes
+
+1. Look for the **Settings** menu located in the top menu bar (just to the right of File and Edit)
+
+1. Click on **Settings** to open the configuration options
+
+    <p align="center">
+    <img src="../assets/gtn_settings_menu.png" alt="Settings Menu" width="500"/>
+    </p>
+
+### 2. Add your Grok.ai API Key in API Keys & Secrets
```

Comment:
xAI account.

I know, even though our API key is GROK_API_KEY, it's actually an xAI account. We might have to change this at some point.. but for now, let's leave the API key as is, but just make sure we're referencing xAI instead of Grok.ai